<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dock_Knot_Style</class>
 <widget class="QDockWidget" name="Dock_Knot_Style">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="cusp-pointed">
    <normaloff/>
   </iconset>
  </property>
  <property name="windowTitle">
   <string>Default Style</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>233</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Curve</string>
       </property>
       <property name="buddy">
        <cstring>spin_handle_length</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spin_handle_length">
       <property name="toolTip">
        <string>Size of the curve control handles</string>
       </property>
       <property name="suffix">
        <string>px</string>
       </property>
       <property name="maximum">
        <double>8192.000000000000000</double>
       </property>
       <property name="value">
        <double>24.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Nodes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Angle</string>
          </property>
          <property name="buddy">
           <cstring>spin_cusp_angle</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spin_cusp_angle">
          <property name="toolTip">
           <string>Minimum angle required to render a cusp</string>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string>Â°</string>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>225.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Distance</string>
          </property>
          <property name="buddy">
           <cstring>spin_cusp_distance</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spin_cusp_distance">
          <property name="toolTip">
           <string>Distance of the cusp tip from the node position</string>
          </property>
          <property name="suffix">
           <string>px</string>
          </property>
          <property name="minimum">
           <double>-8192.000000000000000</double>
          </property>
          <property name="maximum">
           <double>8192.000000000000000</double>
          </property>
          <property name="value">
           <double>32.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Shape</string>
          </property>
          <property name="buddy">
           <cstring>combo_cusp_shape</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="combo_cusp_shape">
          <property name="toolTip">
           <string>Shape style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Edges</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Gap</string>
          </property>
          <property name="buddy">
           <cstring>spin_crossing_gap</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="spin_crossing_gap">
          <property name="toolTip">
           <string>Distance between the ends of a thread passing under another thread</string>
          </property>
          <property name="suffix">
           <string>px</string>
          </property>
          <property name="maximum">
           <double>8192.000000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Slide</string>
          </property>
          <property name="buddy">
           <cstring>slide_edge_slide</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSlider" name="slide_edge_slide">
          <property name="toolTip">
           <string>Position of the crossing in the edge</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
